Fix/doc 4288 (#99)
exoplatform/exo-documentation: docs/Manage-Space.rst

   -  Close: The user cannot request for joining, but only the space
      manager can invite him.

> .. note:: Template, Visbility and registration fields have a description of the choice selected to help you taking right decisions when creating a new space.

Reviewer: ```suggestion
.. note:: Template, Visibility and registration fields have a description of the choice selected to help you choosing the best template for your needs when creating a new space.
```

Reviewer: Also, shouldn't fields be named as they appear on the screenshot to avoid confusion ?

> .. note:: Template, Visbility and registration fields have a description of the choice selected to help you taking right decisions when creating a new space.

Reviewer: ```suggestion
.. note:: Template, Visibility and Registration fields have a description of the choice selected to help you taking right decisions when creating a new space.
```
